tests/tools/bandit/test_ruff_tool.py

```diff
@@ -0,0 +1,18 @@
+from pathlib import Path
+from unittest import TestCase
+from unittest.mock import patch
+
+from prospector.config import ProspectorConfig
+from prospector.finder import FileFinder
+from prospector.tools.ruff import RuffTool
+
+
+class test_ruff_tool:
+    with patch("sys.argv", [""]):
```

Comment:
There are some patch utilities for things like this in [tests/utils.py](https://github.com/prospector-dev/prospector/blob/master/tests/utils.py) - probably you don't need them, but there are various compatibility shims for differing versions of Python over the years so it might make sense to use them anyway. Up to you.

Reply:
Effectively, thanks :-)

Reply:
It's not working:
In the CI I get the error:
```
prospector: error: unrecognized arguments: --benchmark-disable --cov --cov-report=
```
Prospector will get the arguments passed to pytest :-)